### ISSUE TITLE
error in 'egg_base' option: 'src.tmp' does not exist or is not a directory

### ISSUE THREAD

**Issue by dpinol** [pandas-dev/pandas-stubs#26]
https://github.com/VirtusLab/pandas-stubs suggests to install the new stubs with 
```pip install git+https://github.com/pandas-dev/pandas-stubs```

But I get this error. thanks
```
pip install "git+https://github.com/pandas-dev/pandas-stubs"
Collecting git+https://github.com/pandas-dev/pandas-stubs
  Cloning https://github.com/pandas-dev/pandas-stubs to /tmp/pip-req-build-9xsaj4f8
  Running command git clone --filter=blob:none --quiet https://github.com/pandas-dev/pandas-stubs /tmp/pip-req-build-9xsaj4f8
  Resolved https://github.com/pandas-dev/pandas-stubs to commit 2eba4d4e512421927a4ba2e6d0ac7bbd4e934afe
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [4 lines of output]
      /tmp/pip-build-env-02ltw3m2/overlay/lib/python3.10/site-packages/setuptools/config/setupcfg.py:459: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
        warnings.warn(msg, warning_class)
      running egg_info
      error: error in 'egg_base' option: 'src.tmp' does not exist or is not a directory
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

```
note: This error originates from a subprocess, and is likely not a problem with pip.

**Comment by BrenoJesusFernandes** [comment]
Looks like https://github.com/pandas-dev/pandas-stubs/pull/25 fix this.

I tried it on my branch and works fine.
`pip install "git+https://github.com/BrenoJesusFernandes/pandas-stubs.git@feature/poetry"`

**Comment by Dr-Irv** [comment]
Since with #25 we'll use `poetry`, and it seems to address this issue, let's wait until that is merged in, which should happen pretty soon.

**Comment by Dr-Irv** [comment]
`pip install git+https://github.com/pandas-dev/pandas-stubs`  now works since we have merged in #25
